book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals << rental unless @rentals.includes?(rental)
+    rental.book = self
+  end
```

Comment:
Good job by creating the add_rental method to book class 👍🏼 but from the rental class you implemented the rental should receive a book in the initializer so no need to assign the book again I would suggest creating a new rental instance here in this method this will let us assign book and person all at the same time  and same goes for person add_rental method 

`
```ruby
 def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```
`